wordpress-mobile/gutenberg-mobile
# Fix menu item width inside the inserter

Fixes #1906 
Fixes #1907 

Ref to gutenberg: https://github.com/WordPress/gutenberg/pull/20238

To test:

1. Change device font size to the largest
2. Change display zoom to max (if available)
3. Open the app
4. Press the inserter
5. Observe if all the items are displayed properly within a grid

PR submission checklist:
